### DISCUSSION TITLE
[Script request] Clear Linux LXC

### DISCUSSION THREAD

**Discussion by ygsm1** [community-scripts/ProxmoxVE#54] [Request script]
### Application Name

Clear Linux LXC

### Website

https://www.clearlinux.org/downloads.html

### Description

The Clear Linux operating system in LXC template would be really appreciated.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Doesnt have an LXC Template. Only as VM possible.
